docs/video.adoc

```diff
@@ -0,0 +1,75 @@
+== Video Recording
+NOTE: This feature only works when browsers are run in containers. VNC type containers should be used from <<Browser Image information>> section.
+
+Selenoid can capture browser screen and save it to https://en.wikipedia.org/wiki/MPEG-4[MPEG-4] video file using https://en.wikipedia.org/wiki/H.264/MPEG-4_AVC[H264] codec.
+Video recording works by attaching a separate container with https://www.ffmpeg.org/[video capturing software] to running browser container.
+
+To setup this feature you should:
+
+. Pull video recorder image once: `docker pull selenoid/video-recorder`
+
+*Without docker* videos will be stored in `video` directory, relative to binary.
+
+*With docker*:
+
+. Mount host directory for video to `/opt/selenoid/video/` (this is default value for `video-output-dir` selenoid arg in docker)
```

Comment:
just curious when did this path `/opt/selenoid/video/` change?..

when I tested video recording feature, it was required to map host folder with `/output` (at least @vania-pooh said that it should be default)...

@lanwen just to confirm: should we use this path for mapping now?

Reply:
@sskorol this is under discussion. Please consider old values for the moment.

Reply:
@vania-pooh got it, thanks.

Reply:
default value will be `/opt/selenoid/video/` after merge. This can be overridden with `video-output-dir` arg at any time